src/services/automation/editor/automation.js

```diff
@@ -132,7 +132,10 @@
     function load(idTaskType, idScheduledTask) {
       // hack to hide main scroll just for automation
       // TODO fix it #goto issues
-      document.querySelector('html').style.overflow = 'hidden';
+      // Added rule idTaskType !== 0 for automation type for visual issue
```

Comment:
El comentario, al menos a mi, no me deja muy claro que caso es el que se esta tratando. 
Segun me explicaste, entiendo que si idTaskType es 0, es cuando estamos en la pantalla de seleccion de automation type, y si es distinto a 0 estamos en el editor (editando un automation). Es asi?

Por ahi podria dejar claro eso en el comment, o quizas considerar un flag que hago un poco mas explicito esto, por ejemplo: 
```
const isSelectingAutomationType = idTaskType === 0;
if(!isSelectingAutomationType) {
  document.querySelector('html').style.overflow = 'hidden';
}
```
What do you think?